common/friendbot/bin/start

```diff
@@ -3,10 +3,23 @@
 set -e
 set -o pipefail
 
-while ! (curl -sf http://localhost:8001/ | jq --exit-status '.ingest_latest_ledger > 1'); do
+CONF=/opt/stellar/friendbot/etc/friendbot.cfg
+
+# Wait for rpc to be ready if friendbot is configured to use it.
+if grep -q rpc_url "$CONF"; then
+  echo "Waiting for rpc to be available..."
+  while ! (curl -sf -X POST -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0","id":1,"method":"getHealth"}' http://localhost:8003 | jq --exit-status '.result.status == "healthy"' >/dev/null); do
```

Comment:
can remove the h/c `http://localhost:8003` in favor of parsing out the url provided by `rpc_url` in CONF ? Same for the horizon_url reference also.

Reply:
Good call. cbc8bd3baa3f5f06a2b93765ae34f45eb8888e6f